build.gradle

```diff
@@ -25,13 +25,13 @@ task clean(type: Delete) {
 // Define versions in a single place
 ext {
     // Sdk and tools
-    minSdkVersion = 18
+    minSdkVersion = 23
     targetSdkVersion = 24
     compileSdkVersion = 24
     buildToolsVersion = '24.0.2'
```

Comment:
It's better to keep `targetSdkVersion`, `compileSdkVersion`, `buildToolsVersion`, and `supportLibraryVersion` to the same major release, i.e. 25 here.